pallets/parachain-staking/src/lib.rs

```diff
@@ -1339,8 +1339,10 @@ pub mod pallet {
 			more: BalanceOf<T>,
 		) -> DispatchResultWithPostInfo {
```

Comment:
Revert this function to exactly the same with Moonbeam’s original code. 
https://github.com/moonbeam-foundation/moonbeam/blob/4670c614274645139c7ba4d37fa58a1a4e7f2ec3/pallets/parachain-staking/src/lib.rs#L1282


The return is different at the moment.

Reply:
Yes, Moonbeam has indeed modified the return value in the new version.

However, we are currently using the runtime-2302 tag, which corresponds to polkadot-v0.9.38.

https://github.com/moonbeam-foundation/moonbeam/blob/runtime-2302/pallets/parachain-staking/src/lib.rs#L1338

Reply:
okay as long as we use the same cold as runtime-2302